### PR TITLE
Feature/client timeout

### LINKED ISSUE #261: Client timeout
Is it possible to increase request timeout in Turnilo ? Now it returns error "Query error client timeout" after 60 seconds. I tried to increase hardcoded .timeout(60000) in src/client/utils/ajax/ajax.ts. But now it returns "502 Bad Gateway The server returned an invalid or incomplete response" after 2 min.

### PR DESCRIPTION
**Pull request by adrianmroz**
Closes #261

### CODE REVIEW

File: src/server/utils/requester/requester.ts [hunk 1]
```diff
@@ -38,52 +39,39 @@ export interface ProperRequesterOptions {
...
+export function properRequesterFactory(options: ProperRequesterOptions): PlywoodRequester<any> {
+  const { retry, verbose, concurrentLimit } = options;
 
-  return requester;
+  return threadTruthy(
+    createRequester(options),
+    retry && setRetryOptions(retry),
+    verbose && setVerbose,
+    concurrentLimit && setConcurrencyLimit(concurrentLimit)
+  );
```
Comment (piter75) [inline src/server/utils/requester/requester.ts]: It looks much nicer than before 👍

Comment (l2dy) [reply src/server/utils/requester/requester.ts]: I got a `TypeError: f is not a function` in `Object.threadTruthy` with Turnilo 1.12.0-beta.0. Could it be that when `verbose` is `false`, `false` would be passed to threadTruthy() which expects function arguments?

Comment (adrianmroz) [reply src/server/utils/requester/requester.ts]: Fix is on the way in #271. Just few hours :)

**Comment by Laboltus** [comment]
Hello. I'm testing Turnilo 1.12.0 release and now I get Client timeout error after 45 sec. Here is my "clusters" section in config file:

```clusters:
  - name: druid
    type: druid
    host: {BROKER_HOST}:{BROKER_PORT}
    timeout: 300000```

**Comment by adrianmroz** [comment]
Looking into it, I see that not all cluster settings flow through app correctly.

**Comment by adrianmroz** [comment]
Fix in #313

**Comment by Laboltus** [comment]
Thanks, looks like it helped with "client timeout" error. But not after 2 minutes I see the following error:
```
Query error
<html><body><h1>502 Bad Gateway</h1> The server returned an invalid or incomplete response. </body></html>
```

**Comment by adrianmroz** [comment]
Looks like express default timeout.

**Comment by adrianmroz** [comment]
Did some fixes to #313, but not sure about them. It's hard for me to test such case.

**Comment by Laboltus** [comment]
It works now. Thanks! 
JFYI after exceeding the timeout, I still see the error "502 Bad Gateway". Perhaps there should be a more informative message.

**Comment by adrianmroz** [comment]
I agree but I'm not so sure how. I will create issue and track there.

**Pull request merged**